app/models/interaction_search_result.rb

```diff
@@ -1,25 +1,23 @@
 class InteractionSearchResult
```

Comment:
This class was also changed in #190 so we might see some merge conflicts here depending on who merges first.